### PR TITLE
docs: changes subdomain to opensource.docs.scylladb.com

### PR DESCRIPTION
**Pull request by dgarcia360**
docs.scylladb.com will point to https://github.com/scylladb/scylladb-docs-homepage

This pull request changes the domain of this repo to opensource.docs.scylladb.com and moves all the redirects to https://github.com/scylladb/scylladb-docs-homepage/blob/main/docs/_utils/redirects.yaml

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,152 +1,36 @@
...
 
-.. raw:: html
+ScyllaDB Open Source is a distributed NoSQL wide-column database for data-intensive apps that require 
+high performance and low latency.
 
-  <div class="landing__content landing__content">
+You can deploy ScyllaDB Open Source:
 
-.. raw:: html
+* Todo
```
Comment (dgarcia360) [inline docs/index.rst:20]: @annastuchlik Could you please provide the content for the new ScyllaDB Open Source index page?

Comment (annastuchlik) [reply docs/index.rst:20]: We've decided to open two separate PRs for creating the new homepage (step 1) and updating the contents of the existing one (step 2).

**Review by annastuchlik — APPROVED** [review]
LGTM

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please merge.
This merge is urgent as it's part of the process to migrate the OSS docs to opensource.docs.scylladb.com

**Pull request merged**